src/sinks/mod.rs

```diff
@@ -51,7 +51,9 @@ pub mod elasticsearch;
 pub mod file;
 #[cfg(feature = "sinks-gcp")]
 pub mod gcp;
-#[cfg(feature = "sinks-gcp")]
+#[cfg(feature = "sinks-gcp-chronicle")]
+pub mod gcp_chronicle;
+#[cfg(any(feature = "sinks-gcp-chronicle", feature = "sinks-gcp"))]
```

Comment:
```suggestion
#[cfg(feature = "sinks-gcp")]
```

Does chronicle depend on gcs_common? That suprises me.

Reply:
Yes it looks like it: https://github.com/vectordotdev/vector/blob/master/src/sinks/gcp/chronicle_unstructured.rs#L35

Reply:
Ah, it seems like the reference types there should probably be in `gcp_common` but that's definitely out-of-scope of this PR 👍